docs/collator-instructions.md

```diff
@@ -0,0 +1,177 @@
+# How to set up an InvArch Collator
+
+Make sure you are logged in as a user with root privileges, otherwise:
+
+``sudo su -``
+
+Create a new service user to run your collator service:
+`sudo useradd --no-create-home --shell /usr/sbin/nologin tinkernet-collator`
+
+Install some required stuff:
+
+``curl https://sh.rustup.rs -sSf | sh``
+(choose option 1 - Proceed with installation (default))
+You may need to restart your system before the next steps.
+```
+rustup install nightly-2021-11-02
+rustup target add wasm32-unknown-unknown --toolchain nightly-2021-11-02
```

Comment:
These should not be needed as they would install the incorrect version anyway, as per the last release rustup should auto install all components for the version specified in the rust-toolchain.toml